user-service/src/main/java/org/java/mentorship/user/repository/mapper/SessionMapper.java

```diff
@@ -0,0 +1,29 @@
+package org.java.mentorship.user.repository.mapper;
+
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Options;
+import org.apache.ibatis.annotations.Select;
+import org.java.mentorship.contracts.user.dto.Session;
+
+import java.util.List;
+import java.util.Optional;
+
+@Mapper
+public interface SessionMapper {
```

Comment:
You can however rename this class to "SessionRepository"
So we don't confuse it with the Mapper design pattern